saleor/dashboard/customer/views.py

```diff
@@ -17,7 +17,8 @@ def customer_list(request):
         .select_related('default_billing_address', 'default_shipping_address')
         .annotate(
             num_orders=Count('orders', distinct=True),
-            last_order=Max('orders', distinct=True)))
+            last_order=Max('orders', distinct=True))
+    )
```

Comment:
Why?

Reply:
As in commit message. Staff can be also a customer. Now we have a separate view with only staff members. If sb would like to see other user Order history, he will not have to search in two places.
- Customers - list of all users
- Staff members - list of users which are staff members